src/lib/Tstorish.sol

```diff
@@ -114,11 +91,59 @@ contract Tstorish {
         }
     }
 
+    function _setTstorish(uint256 slot, uint256 value) internal {
+        bool tstoreInitialSupport = _tstoreInitialSupport;
+
+        assembly ("memory-safe") {
+            for { } 1 { } {
```

Comment:
should add a comment explaining that the for block is used for early function returns